src/iocore/net/quic/QUICStreamManager.cc

```diff
@@ -89,15 +102,15 @@ QUICStreamManager::find_stream(QUICStreamId stream_id)
   return nullptr;
 }
 
-QUICConnectionErrorUPtr
-QUICStreamManager::create_stream(QUICStreamId stream_id)
+QUICStream *
+QUICStreamManager::create_stream(QUICStreamId stream_id, QUICConnectionError &err)
```

Comment:
This is just to avoid calling `find_stream` twice when creating a stream, no much effect on performance.